src/components/utils/field-constants.js

```diff
@@ -8,8 +8,8 @@
 export const EQUIPMENT_ID = 'equipmentId';
 export const EQUIPMENT_NAME = 'equipmentName';
 export const LOAD_TYPE = 'loadType';
-export const ACTIVE_POWER = 'activePower';
-export const REACTIVE_POWER = 'reactivePower';
+export const ACTIVE_POWER = 'activePowerSetpoint';
+export const REACTIVE_POWER = 'reactivePowerSetpoint';
```

Comment:
There is already in the file theses entries : 
`export const ACTIVE_POWER_SET_POINT = 'activePowerSetpoint';`
`export const REACTIVE_POWER_SET_POINT = 'reactivePowerSetpoint';`